tests/test_CoordGeomPy.py

```diff
@@ -1,6 +1,32 @@
 from CoordGeomPy import CoordGeomPy
+from CoordGeomPy import dist_pll_lines_2d
+import numpy as np
 import pytest
 
+def test_dist_pll_lines_2d():
+    """Finding the distance between two parallel lines"""
+
+    m = 2
+    b1 = 4
+    b2 = -1
+    d = 2.23606797749979
+
+    # data type errors 
+    assert isinstance(m, (int, float)), "Slope is neither an integer nor a float"
+    assert isinstance(b1, (int, float)), "Intercept of line 1 is neither an integer nor a float"
+    assert isinstance(b2, (int, float)), "Intercept of line 2 is neither an integer nor a float"
+    assert isinstance(d, (int, float)), "Result is wrong data type.."
+
+    # result value approximation
+    assert CoordGeomPy.dist_pll_lines_2d(m, b1, b2) == d
+    assert round(CoordGeomPy.dist_pll_lines_2d(2, 4, -1), 2) == 2.24
+
+    # ivalid arguments check
+    try: 
+        dist_pll_lines_2d(m)
+    except: 
+        print("Missing arguments, please add all the inputs required!")
```

Comment:
I could be totally out to lunch here but will the way this is written work with pytests? 

It might have to look something like this? 
`with pytest.raises(TypeError):
        list_object = ["Pythons", "are", "non", "venomous"]
        plot_words(list_object)`

See the notes on testing here: https://py-pkgs.org/05-testing

Reply:
I did test it out on a local test copy package and it did work but based on Florencia's slides today I think exception handling should be included in the source code file. So I'll move this section there.